### ISSUE TITLE
[BUG]: HiDive not autheticating. The password is showing wrong despite being correct password and loginin on Browser

### ISSUE THREAD

**Issue by pratyushpati11** [anidl/multi-downloader-nx#586]
### Program version

4.4.3

### Operating System

Windows

### Type

GUI

### Service

Hidive

### Command used

Autheticate

### Show ID

NA

### Episode

NA

### Console Output

```Shell
[ERROR] Code 33 (InvalidPassword): Invalid Password Please try again. If the problem persists, please contact help@hidive.com.

[ERROR] Authentication failed!
[ERROR] Code 33 (InvalidPassword): Invalid Password Please try again. If the problem persists, please contact help@hidive.com.

[ERROR] Authentication failed!
```

### Additional Information

Not able to open HiDive even when the password is right and working on Browser.

**Comment by Jaynator495** [comment]
I wasn't able to replicate this oddly. It authenticates fine for me. What region are you in? And what command are you using to authenticate?

**Comment by pratyushpati11** [comment]
> I wasn't able to replicate this oddly. It authenticates fine for me. What region are you in? And what command are you using to authenticate?

I use UK region and then USA. I used the GUI so just email and password.Oddly this has worked fine for me previously.Also same region worked on Browser.

**Comment by pratyushpati11** [comment]
It used to Autheticate fine few months ago.

**Comment by pratyushpati11** [comment]
Is this VPN issue? But I have used the same VPN before and it worked.

**Comment by Jaynator495** [comment]
> Is this VPN issue? But I have used the same VPN before and it worked.

I don't believe so, from what I can tell, it seems that new accounts (or subscribed using the new API? Not sure) are partially broken in the old API - Likely just authentication. Will likely have to implement the API in it's entirety so new users can use it, but I'll add a flag to use the old API

**Comment by pratyushpati11** [comment]
> > Is this VPN issue? But I have used the same VPN before and it worked.
> 
> I don't believe so, from what I can tell, it seems that new accounts (or subscribed using the new API? Not sure) are partially broken in the old API - Likely just authentication. Will likely have to implement the API in it's entirety so new users can use it, but I'll add a flag to use the old API

Yeah I re subsscribed while using new API .

**Comment by pratyushpati11** [comment]
@Jaynator495 Here is the answer https://github.com/anidl/multi-downloader-nx/issues/585#issuecomment-1987141746

**Comment by Jaynator495** [comment]
> @Jaynator495 Here is the answer [#585 (comment)](https://github.com/anidl/multi-downloader-nx/issues/585#issuecomment-1987141746)

Responding to this a little late, but as far as I can tell, authentication is *not* compatible between the APIs, so I can't use the new API to authenticate and then fallback to the old API.

**Comment by pratyushpati11** [comment]
> > @Jaynator495 Here is the answer [#585 (comment)](https://github.com/anidl/multi-downloader-nx/issues/585#issuecomment-1987141746)
> 
> Responding to this a little late, but as far as I can tell, authentication is _not_ compatible between the APIs, so I can't use the new API to authenticate and then fallback to the old API.

Ok .So is there any solution for me? I can't login with GUI.

**Comment by Jaynator495** [comment]
> > > @Jaynator495 Here is the answer [#585 (comment)](https://github.com/anidl/multi-downloader-nx/issues/585#issuecomment-1987141746)
> > 
> > 
> > Responding to this a little late, but as far as I can tell, authentication is _not_ compatible between the APIs, so I can't use the new API to authenticate and then fallback to the old API.
> 
> Ok .So is there any solution for me? I can't login with GUI.

Not yet as far as I know. I am working on implementing the new API though, but it is taking time since it's pretty much a complete rewrite of the hidive portions.

**Comment by pratyushpati11** [comment]
> > > > @Jaynator495 Here is the answer [#585 (comment)](https://github.com/anidl/multi-downloader-nx/issues/585#issuecomment-1987141746)
> > > 
> > > 
> > > Responding to this a little late, but as far as I can tell, authentication is _not_ compatible between the APIs, so I can't use the new API to authenticate and then fallback to the old API.
> > 
> > 
> > Ok .So is there any solution for me? I can't login with GUI.
> 
> Not yet as far as I know. I am working on implementing the new API though, but it is taking time since it's pretty much a complete rewrite of the hidive portions.

Ok.Thanks for answering.

**Comment by AnimeDL** [comment]
Released version 4.5.1 which adds support for Hidive's new API, you can use it with `--hdapi new` (or by adding `hdapi: "new"` to the cli-defaults.yml file).
